demos/nextjs-sdk-integration-test/src/pages/api/get-templates.tsx

```diff
@@ -0,0 +1,8 @@
+import { EtherealPulse, TemplateDTO } from "@ethereal-pulse/typescript-sdk";
+
+export default async function handler(req: any, res: any) {
+  const ethPulseSDK = new EtherealPulse('aadf9195-fe77-4089-b9af-3fdf867446f6');
```

Comment:
Could you update to use a env variable based api-key in the project references instead a hard coded option?

Reply:
done.